flatland/trajectories/trajectories.py

```diff
@@ -0,0 +1,558 @@
+import ast
+import importlib
+import os
+import uuid
+from pathlib import Path
+from typing import Optional, Any, Tuple
+
+import click
+import pandas as pd
+import tqdm
+from attr import attrs, attrib
+
+from flatland.core.env_observation_builder import ObservationBuilder
+from flatland.env_generation.env_generator import env_generator
+from flatland.envs.persistence import RailEnvPersister
+from flatland.envs.rail_env import RailEnv
+from flatland.envs.rail_env_action import RailEnvActions
+
+DISCRETE_ACTION_FNAME = "event_logs/ActionEvents.discrete_action.tsv"
+TRAINS_ARRIVED_FNAME = "event_logs/TrainMovementEvents.trains_arrived.tsv"
+TRAINS_POSITIONS_FNAME = "event_logs/TrainMovementEvents.trains_positions.tsv"
+SERIALISED_STATE_SUBDIR = 'serialised_state'
+
+
+class Policy:
+    def act(self, handle: int, observation: Any, **kwargs) -> RailEnvActions:
```

Comment:
@manuschn  Not sure we want/need the handle as well? The policy's acting should be based on the observation only and not depend on the handle?